src/DonationForms/V2/resources/components/Onboarding/Dialogs/FeatureNoticeDialog.tsx

```diff
@@ -27,6 +28,14 @@ export default function FeatureNoticeDialog({isUpgrading, isEditing, handleClose
         }
     }
 
+    // @note the <Button/> component does not support the `className` prop.
+    const upgradeButtonStyles = {
+        width: '100%',
+        marginTop: 'var(--givewp-spacing-6)',
+        marginBottom: 'var(--givewp-spacing-4)',
+        backgroundColor: 'var(--wp-blue-blue-50)'
+    }
```

Comment:
Can we add the `className` prop to the `Button` component without any issues? I tested it locally and apparently it worked as expected. This way, we can avoid mixing inline styles with CSS modules in the same file.

Reply:
@pauloiankoski I think that would be a good refactor, but I'd rather not change shared components in this PR.